public/locales/en/translation.json

```diff
@@ -104,8 +104,7 @@
     "form": {
       "createListing": "Create agent listing",
       "edit": "Edit",
-      "firstName": "Agent first name",
-      "firstNameRequired": "Please provide a first name",
+      "firstName": "Agent first name (optional)",
```

Comment:
There is a `ui.optional` string that can be used as needed, better to use that than change this string.

Reply:
I'll put a code example in the form below.